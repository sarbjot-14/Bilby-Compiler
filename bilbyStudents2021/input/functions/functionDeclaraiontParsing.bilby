
func int weight() {

	return 555 ;
}

main{

	imm ans := weight();
	print ans $n;

}
