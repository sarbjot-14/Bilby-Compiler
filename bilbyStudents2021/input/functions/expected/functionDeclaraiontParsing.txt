[ 186 163 627699 33 ]
[ 186 33 ]
33
